PatchLevelCompatibilityTest failures

master (commit aeaec0a615a9f7d2fe6571f8c5858adeab8fa734)

Failed on nightly build : http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-3.x-nightly/128/testReport/

PatchLevelCompatibilityTest.testAll_V37_Versions
PatchLevelCompatibilityTest.testAll_V39_Versions
PatchLevelCompatibilityTest.testAll_V310_Versions
PatchLevelCompatibilityTest.testAll_V38_Versions

**Stacktrace**
```
com.hazelcast.test.starter.GuardianException: java.lang.ClassNotFoundException: com.hazelcast.core.Cluster
	at com.hazelcast.test.starter.ProxyInvocationHandler.proxyReturnObject(ProxyInvocationHandler.java:168)
	at com.hazelcast.test.starter.ProxyInvocationHandler.invoke(ProxyInvocationHandler.java:70)
	at com.sun.proxy.$Proxy78.getCluster(Unknown Source)
	at com.hazelcast.test.HazelcastTestSupport.getClusterSize(HazelcastTestSupport.java:1099)
	at com.hazelcast.test.HazelcastTestSupport.assertClusterSize(HazelcastTestSupport.java:1090)
	at com.hazelcast.test.HazelcastTestSupport$11.run(HazelcastTestSupport.java:1120)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1303)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:1320)
	at com.hazelcast.test.HazelcastTestSupport.assertClusterSizeEventually(HazelcastTestSupport.java:1117)
	at com.hazelcast.test.HazelcastTestSupport.assertClusterSizeEventually(HazelcastTestSupport.java:1105)
	at com.hazelcast.test.starter.test.PatchLevelCompatibilityTest.testAllGivenVersions(PatchLevelCompatibilityTest.java:95)
	at com.hazelcast.test.starter.test.PatchLevelCompatibilityTest.testAll_V37_Versions(PatchLevelCompatibilityTest.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: com.hazelcast.core.Cluster
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at com.hazelcast.test.starter.HazelcastProxyFactory.constructWithJdkProxy(HazelcastProxyFactory.java:257)
	at com.hazelcast.test.starter.HazelcastProxyFactory.proxyObjectForStarter(HazelcastProxyFactory.java:197)
	at com.hazelcast.test.starter.ProxyInvocationHandler.proxyReturnObject(ProxyInvocationHandler.java:164)
	... 23 more
```

--- a/hazelcast/src/test/java/com/hazelcast/test/starter/test/PatchLevelCompatibilityTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/test/starter/test/PatchLevelCompatibilityTest.java
@@ -32,6 +32,7 @@ import static com.hazelcast.test.HazelcastTestSupport.assertClusterSizeEventuall
 import static com.hazelcast.test.HazelcastTestSupport.smallInstanceConfig;
 import static org.junit.Assert.assertEquals;
 
+@Ignore("https://github.com/hazelcast/hazelcast/issues/15021")
 @RunWith(HazelcastSerialClassRunner.class)
 @Category(NightlyTest.class)
 public class PatchLevelCompatibilityTest {
@@ -73,7 +74,6 @@ public class PatchLevelCompatibilityTest {
         testAllGivenVersions(versions);
     }
 
-    @Ignore("https://github.com/hazelcast/hazelcast/issues/15021")
     @Test
     public void testMap_whenMixed_V37_Cluster() {
         String[] versions = new String[]{"3.7.4", "3.7.5"};

